notices.json

```diff
@@ -110,5 +110,23 @@
         "actionParam": "https://mattermost.com/blog/collapsed-reply-threads-beta/"
       }
     }
+  },
+  {
+    "id": "v6.0_user_introduction",
+    "conditions": {
+      "audience": "all",
+      "clientType": "all",
+      "serverVersion": [">=6.0"],
+      "displayDate": ">= 2021-10-13T00:00:00Z"
```

Comment:
@amyblais we should prioritize shipping this notice since it should appear for everyone immediately after their admin does the v6.0 upgrade